test/contracts/staking/staking-factory.test.ts

```diff
@@ -37,10 +37,11 @@ import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers'
 // Wires up Waffle with Chai
 chai.use(solidity)
 
-const EPOCH_DURATION = 60
+const EPOCH_DURATION = BigNumber.from(60)
```

Comment:
suggestion: Use the short hand for `BigNumber` e.g. `60n`

This may be a personal style choice, but as I've used the shorthand everywhere in the Bonds, it might be a de-facto standard 🤷

Reply:
Thats the shorthand for the javascript built-in`bigint` vs. ethers exported`BigNumber` though, right? Will I switch to using `bigint` everywhere in these tests?

Reply:
Does your code require `BigNumber` or `BigNumberish`?

Much of the TypeChain code seem to cater for `BigNumberish`, which allows using `bigint`, making the code look pretty nice (what you've got there with the casting looks a lot like Java from many years ago :wink)